apis/common/v1/policies.go

```diff
@@ -16,23 +16,39 @@ limitations under the License.
 
 package v1
 
-// A ManagementPolicy determines how should Crossplane controllers manage an
-// external resource.
-// +kubebuilder:validation:Enum=FullControl;ObserveOnly;OrphanOnDelete
-type ManagementPolicy string
+// ManagementPolicy determines how should Crossplane controllers manage an
+// external resource through an array of ManagementActions.
+type ManagementPolicy []ManagementAction
+
+// A ManagementAction represents an action that the Crossplane controllers
+// can take on an external resource.
+// +kubebuilder:validation:Enum=Observe;Create;Update;Delete;LateInitialize;*
+type ManagementAction string
 
 const (
-	// ManagementFullControl means the external resource is fully controlled
-	// by Crossplane controllers, including its deletion.
-	ManagementFullControl ManagementPolicy = "FullControl"
+	// ManagementActionObserve means that the managed resource status.atProvider
+	// will be updated with the external resource state.
+	ManagementActionObserve ManagementAction = "Observe"
+
+	// ManagementActionCreate means that the external resource will be created
+	// when using the managed resource spec.initProvider and spec.forProvider.
```

Comment:
nit: do we need `when` ? also for update.
```suggestion
	// using the managed resource spec.initProvider and spec.forProvider.
```